beesdoo_product/views/beesdoo_product.xml

```diff
@@ -110,6 +114,32 @@
         </field>
     </record>
 
+    <record model="ir.ui.view" id="beesdoo_product_hazard_form">
+        <field name="name">bees.product.hazard.form</field>
+        <field name="model">beesdoo.product.hazard</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="name"/>
+                    <field name="type"/>
+                    <field name="active"/>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="beesdoo_product_hazard_tree">
+        <field name="name">bees.product.hazard.tree</field>
+        <field name="model">beesdoo.product.hazard</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <field name="name"/>
+                <field name="type"/>
+                <field name="active"/>
```

Comment:
if you want to keep no need to have it in a tree view as Odoo hide inactive records by the default

Reply:
I don't want to keep the active field